js_modules/dagit/packages/core/src/ticks/tests/SensorDryRunDialog.test.tsx

```diff
@@ -73,6 +73,23 @@ describe('SensorDryRunTest', () => {
     });
   });
 
+  it('allows you to test again', async () => {
+    await act(async () => {
+      render(<Test mocks={[SensorDryRunMutationError]} />);
+    });
```

Comment:
I think since you're using `MockedProvider`, you don't need the render to be async. (At least, I haven't been using it and it hasn't been a problem.)